charts/log-generator/templates/deployment.yaml

```
@@ -27,7 +27,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

use appVersion in Chart.yaml without v prefix

```suggestion
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" .Chart.AppVersion) }}"
```

---

PS: Maybe the image tag should also not have prefix

---

I'm not sure I understand this.
```
appVersion: v0.5.0
```

appVersion has prefix

---

hope it is clear now, with the comment in the new PR?